client/src/features/movie-list/component.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import './component.less'
+import MovieItem from '../movie-item'
+import getFilms from'./utils';
+
+class MovieList extends React.Component {
+  
+  constructor(props) {
```

Comment:
no need to describe constructor here, just init state as class field:
``` js
state = {films: []};
```
please use default [ ] for arrays